test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -1131,6 +1135,19 @@ protected static void wipeAllIndices(boolean preserveSecurityIndices) throws IOE
             }
             final Request deleteRequest = new Request("DELETE", Strings.collectionToCommaDelimitedString(indexPatterns));
             deleteRequest.addParameter("expand_wildcards", "open,closed,hidden");
+
+            // If system index warning, ignore but log
+            deleteRequest.setOptions(RequestOptions.DEFAULT.toBuilder().setWarningsHandler(warnings -> {
```

Comment:
I know we talked about this yesterday in the team meeting, but I don't remember if we talked about excluding the `triggered-watches` data stream from this DELETE request. Is that not an option?

Reply:
I don't remember discussing that but I was a bit hesitant to do that as I _think_ there's logic that reads that index and may affect the execution / retrieval of watches if it's left over between tests.

Perhaps @masseyke can weigh in here if he thinks it's safe to keep between tests?

Reply:
Did a bit more digging and yes `.triggered-watches` hanging around between tests does affect watch execution (see https://github.com/elastic/elasticsearch/blob/8efd08b019b9160f5e703520c6ce3b6a9f92cfbd/x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/execution/TriggeredWatchStore.java#L121) so I think it's best we tidy it up.

@nielsbauman Are you happy with that conclusion?